### PR TITLE
Add unittests for organization-private problems

### PR DESCRIPTION
**Pull request by Ninjaclasher**
Refs #1440 and #1464.

### CODE REVIEW

File: judge/models/tests/test_problem.py [hunk 1]
```diff
@@ -156,6 +172,62 @@ def test_organization_private_problem_methods(self):
...
+                'is_editable_by': self.assertFalse,
+            },
+            'anonymous': {
+                'is_accessible_by': self.assertFalse,
+                'is_editable_by': self.assertFalse,
+            },
+        }
+        self._test_object_methods_with_users(self.organization_admin_private_problem, data)
+
+    def test_organization_admin_problem_methods(self):
```
Comment (kiritofeng) [inline judge/models/tests/test_problem.py:210]: Shouldn't you also test an organization admin on this problem?

Comment (Ninjaclasher) [reply judge/models/tests/test_problem.py:210]: The `problem_organization` organization has `normal` and `staff_problem_edit_public` as organization admins (unless you mean a different organization admin (?)).

Comment (kiritofeng) [reply judge/models/tests/test_problem.py:210]: Ah oops I didn't read that correctly.

Trying this with the admin of another organization would make for a good unit test.

Comment (Ninjaclasher) [reply judge/models/tests/test_problem.py:214]: Done

**Pull request merged**